doc/using_pytorch.rst

```diff
@@ -118,10 +118,21 @@ to a certain filesystem path called ``model_dir``. This value is accessible thro
 After your training job is complete, SageMaker will compress and upload the serialized model to S3, and your model data
 will be available in the S3 ``output_path`` you specified when you created the PyTorch Estimator.
 
+If you are using Elastic Inference, you must convert your models to the TorchScript format and use ``torch.jit.save`` to save the model.
+For example:
+
+.. code:: python
+
+    import os
+    import torch
+
+    model_path = os.path.join(model_dir, "model.pt")
+    model = torch.jit.load(model_path)
```

Comment:
Is this supposed to be an example of saving or loading? I'd think this would be where the example would go that traces the model on an example input to get a scriptmodule and then calls `torch.jit.save` to save it.

Reply:
This is meant to be a snippet for saving the model, modified.